static/app/views/discover/table/cellAction.tsx

```diff
@@ -83,6 +90,9 @@ export function updateQuery(
     }
     // these actions do not modify the query in any way,
     // instead they have side effects
+    case Actions.COPY_TO_CLIPBOARD:
+      navigator.clipboard.writeText(stringifyValue(value));
```

Comment:
This can throw, I would expect we extract this out to a function similar to the react hook for copying and emit error messages accordingly.

https://github.com/getsentry/sentry/blob/df335b5df8c6f2a364d768d69b0f20e19f4d99bb/static/app/utils/useCopyToClipboard.tsx#L47